### PR TITLE
Remove Poppins font

### PR DESCRIPTION
**Pull request by wittejm**
Use Inter font instead of Poppins and remove dependency.

### CODE REVIEW

**Comment by wittejm** [comment]
[sc-835]

File: package-lock.json [hunk 1]
```diff
@@ -6,6 +6,7 @@
   "packages": {
     "": {
       "version": "1.2.6",
+      "hasInstallScript": true,
```
Comment (wittejm) [inline package-lock.json:9]: Oh I think I need to revert this. But what's the plan for avoiding checking in this change going forward? I'm actually not sure what it's about.

**Review by cainlevy — APPROVED** [review]

File: package-lock.json (hunk 1 above)
Comment (cainlevy) [reply package-lock.json:9]: i'm not familiar with this, but it seems harmless

**Comment by gabrielsaruhashi** [comment]
Congrats on your first PR 🎉 !

**Pull request merged**